client/core/defs.h

```diff
@@ -83,7 +79,13 @@ namespace amnezia
 
         // Api errors
         ApiConfigDownloadError = 1100,
-        ApiConfigAlreadyAdded = 1101
+        ApiConfigAlreadyAdded = 1101,
+
+        // QFile errors
+        OpenError = 1200,
+        ReadError = 1201,
+        PermissionsError = 1202,
+        UnspecifiedError = 1203,
```

Comment:
The documentation does not contain them in https://docs.amnezia.org/troubleshooting/error-codes/
I face 1200 and don't know now how can I resolve it

Reply:
it helped to restart the amnezia-client and try to setup the vpn on vps one more time.